third_party/terraform/tests/resource_compute_instance_from_template_test.go

```diff
@@ -125,6 +125,50 @@ func TestAccComputeInstanceFromTemplate_overrideScratchDisk(t *testing.T) {
 	})
 }
 
+func TestAccComputeInstanceFromTemplate_012_removableFields(t *testing.T) {
+	t.Parallel()
+
+	var instance compute.Instance
+	instanceName := fmt.Sprintf("terraform-test-%s", acctest.RandString(10))
+	templateName := fmt.Sprintf("terraform-test-%s", acctest.RandString(10))
+	resourceName := "google_compute_instance_from_template.inst"
+
+	// First config is a basic instance from template, second tests the empty list syntax
+	config1 := testAccComputeInstanceFromTemplate_012_removableFieldsTpl(templateName) +
+		testAccComputeInstanceFromTemplate_012_removableFields1(instanceName)
+	config2 := testAccComputeInstanceFromTemplate_012_removableFieldsTpl(templateName) +
+		testAccComputeInstanceFromTemplate_012_removableFields2(instanceName)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckComputeInstanceFromTemplateDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config1,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckComputeInstanceExists(resourceName, &instance),
+
+					resource.TestCheckResourceAttr(resourceName, "service_account.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "service_account.0.scopes.#", "3"),
+				),
+			},
+			{
```

Comment:
Import steps instead of check resource exists/attr?

Reply:
instance_from_template isn't importable :(